content/Arduino Cloud/Web Editor/If-your-board-is-not-detected-by-Arduino-Web-Editor.md

```diff
@@ -0,0 +1,37 @@
+---
+title: "If your board is not detected by Arduino Web Editor"
+id: 360018131160
+---
+
+If your board is not detected by Arduino Web Editor, a red cross icon will be displayed in the board selector.
```

Comment:
```suggestion
If the selected board in Arduino Web Editor could not be detected, the board selector will display a red cross.
```

Slight clarification: "My board" (as in a physical board that I want to use with the Web Editor) may be detected, but if a different board is selected in the board selector (and that board is not being detected), the red cross will appear.